arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -1157,12 +1161,22 @@ arangodb::Result IResearchAnalyzerFeature::emplaceAnalyzer( // emplace
 
     erase = false;
   } else if (!equalAnalyzer(*analyzer, type, properties, features)) { // duplicate analyzer with different configuration
-    return arangodb::Result(
-      TRI_ERROR_BAD_PARAMETER,
-      "name collision detected while registering an arangosearch analizer name '" + std::string(name) +
-      "' type '" + std::string(type) + "' properties '" + properties.toString() +
-      "', previous registration type '" + std::string(analyzer->type()) +
-      "' properties '" + analyzer->properties().toString() + "'");
+    std::ostringstream errorText;
+    errorText << "name collision detected while registering an arangosearch analyzer name '" << name
+      << "' type '" << type << "' properties '" << properties.toString()
+      << "' features '"; 
+    for(auto feature : features) {
+      errorText << feature->name() << " ";
+    }   
+    errorText << "', previous registration type '" << analyzer->type()
+      << "' properties '" << analyzer->properties().toString() << "'"
+      << " features '";
+    for(auto feature : analyzer->features()) {
+      errorText << feature->name() << " ";
+    }
+    errorText << "'";
+    return arangodb::Result(TRI_ERROR_BAD_PARAMETER, errorText.str());
```

Comment:
wouldn't this rather be an error with code `TRI_ERROR_ARANGO_CONFLICT`?

Reply:
Description for this error says: "Will be raised when updating or deleting a document and a conflict has been detected."
But at this point we have no conflicts with documents, Most propbably this situation means that user actually have misspelled some of analyzer properties  (e.g used the same name twice)

Reply:
Ok. For me "conflict" means "conflict", so I would have returned `TRI_ERROR_ARANGO_CONFLICT`. However, I can live with any other error code just fine.